code/modules/reagents/machinery/chem_master.dm

```diff
@@ -162,6 +162,12 @@ var/global/list/pillIcon2Name = list("oblong purple-pink", "oblong green-white",
 		src.updateUsrDialog()
 		return 1
 
+	else if(istype(B, /obj/item/weapon/reagent_containers/pill/time_release))
```

Comment:
Is there any reason not to give pills in general the ability to be renamed?

Reply:
Yeah I guess this could go for all pills